config/database.yml

```diff
@@ -1,25 +1,17 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
```

Comment:
Just tested local. It is easy to setup. Rails raised an error only on missing `host` variable:

```yml
host: localhost
```

Reply:
Thank you so much for having a look @workgena, I'll include this missing variable!

Reply:
Updated in 0739d65f9717f0d5f736e661fd02d980a8c07872!